asapdiscovery-data/asapdiscovery/data/selectors/mcs_selector.py

```diff
@@ -114,12 +114,13 @@ def _select(
             for complex in complexes:
                 complex_mol = complex.ligand.to_oemol()
                 # MCS search
-                mcs = next(iter(mcss.Match(complex_mol, True)))
-                sort_args.append((mcs.NumBonds(), mcs.NumAtoms()))
-
+                try:
+                    mcs = next(iter(mcss.Match(complex_mol, True)))
+                    sort_args.append((mcs.NumBonds(), mcs.NumAtoms()))
+                except StopIteration:  # no match found
```

Comment:
Do we have test coverage for this except?

Reply:
we do now :)